bolero/src/lib.rs

```diff
@@ -26,9 +26,13 @@ cfg_if::cfg_if! {
 
 /// Re-export of [`bolero_generator`]
 pub mod generator {
+    // TODO: remove the use of prelude::* for the next major release
```

Comment:
I'd prefer to keep this here for backward compatibility. There's quite a few harnesses out there that this would break.

Reply:
```suggestion
```

Reply:
Makes sense, my hope had been they'd just have to update to use `bolero::gen_with` and would end up with cleaner code :)